test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="librarytest/php-fpm-hello-web:fcgi"
```

Comment:
I'm meh on `cname` (that seems sane and reasonable unless you really feel like expanding it to a longer name), but the underscore here for sure. :+1: